commands/api.go

```diff
@@ -250,9 +251,9 @@ func apiCommand(cmd *Command, args *Args) {
 		}
 
 		if includeHeaders {
-			fmt.Fprintf(out, "%s %s\r\n", response.Proto, response.Status)
-			response.Header.Write(out)
-			fmt.Fprintf(out, "\r\n")
+			_, _ = fmt.Fprintf(out, "%s %s\r\n", response.Proto, response.Status)
+			utils.Check(response.Header.Write(out))
+			_, _ = fmt.Fprintf(out, "\r\n")
```

Comment:
This PR adds some error handling that is unrelated to the change proposed in this PR:

1. The additions make your PR harder to review;
2. Hub doesn't generally perform error handling around writing to stdout/stderr. Even though error handling is a good rule of thumb in general, I think that the lack of error handling here is fine for now, because it's not presenting problems for anyone.

Reply:
I did wonder if it would be received as cleaning up things, or noise, which is why they were in separate commits so they could be reverted or removed

I'm hearing they fall more into the "noise" category so I'll remove them